models/settings/fields.yaml

```diff
@@ -27,6 +27,14 @@ tabs:
             commentAbove: rainlab.user::lang.settings.login_attribute_comment
             type: radio
             tab: rainlab.user::lang.settings.signin_tab
+            
+        # Update requires Password
+        update_requires_password:
+            span: right
+            label: 'User Settings'
+            commentAbove: 'Settings update requires current user password'
```

Comment:
@cbacon93 Please change `label` and `commentAbove` to use Lang file references so that people can translate these fields for other languages.